src/main/java/com/muzima/view/forms/FormsActivity.java

```diff
@@ -153,6 +153,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
                     Toast.makeText(this, "Already fetching forms, ignored the request", Toast.LENGTH_SHORT).show();
                     return true;
                 }
+                if (hasFormsWithData()) {
+                    Toast.makeText(this, "There is existing form data for selected form(s). Finish Incomplete data and sync Complete data to Server first before downloading selected forms", Toast.LENGTH_SHORT).show();
```

Comment:
Make the code fold to the next line when the code is too long. And probably put this text in the strings.xml?